apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GoogleLogger.cs

```diff
@@ -226,5 +235,39 @@ internal string GetTraceName()
         /// </summary>
         internal IEnumerable<ILogEntryLabelProvider> GetLabelProviders() =>
             _serviceProvider?.GetService<IEnumerable<ILogEntryLabelProvider>>();
+
+        /// <summary>
+        /// For diagnostic purposes. Builds and returns the URL where the entries logged by
+        /// this <see cref="GoogleLogger"/> can be seen on the GCP Stackdriver Logging Console.
+        /// </summary>
+        public Uri GetGcpConsoleLogsUrl()
+        {
+            IList<string> parameters = new List<string>
+            {
+                $"resource={_loggerOptions.MonitoredResource.Type}",
+                $"minLogLevel={(int)_loggerOptions.LogLevel.ToLogSeverity()}",
+                $"logName={_fullLogName}",
+                _logTarget.Kind == LogTargetKind.Project ? $"project={_logTarget.ProjectId}" :
+                _logTarget.Kind == LogTargetKind.Organization ? $"organizationId={_logTarget.OrganizationId}" :
+                throw new InvalidOperationException($"Unrecognized LogTargetKind: {_logTarget.Kind}")
+            };
+
+            return new UriBuilder(GcpConsoleLogsBaseUrl)
+            {
+                Query = string.Join("&", parameters)
+            }.Uri;
+        }
+
+        private void WriteCreationDiagnostics()
+        {
+            // Explicitly not catching exceptions.
+            // This should only be activated for diagnostics purposes so in that case
+            // we shouldn't throw exceptions.
+            if (_loggerOptions.LoggerDiagnosticsOutput != null)
```

Comment:
Just use the null conditional operator?

```csharp
_loggerOptions.LoggerDiagnosticsOutput?.WriteLine($"{DateTime.UtcNow} - GoogleLogger created. Logs written at: {GetGcpConsoleLogsUrl()}");
_loggerOptions.LoggerDiagnosticsOutput?.Flush();
```

I'd be tempted to explicitly use string.Format and use the invariant culture as well, so that the date always gets formatted the same way. Ideally, do that with `yyyy-MM-dd'T'HH:mm:ss` as the format, too...